torch/quantization/_numeric_suite.py

```diff
@@ -178,3 +214,70 @@ def compare_model_stub(float_model, q_model, module_swap_list, data, Logger=Logg
     ob_dict = {}
     _get_observer_dict(q_model, ob_dict, "logger")
     return ob_dict
+
+
+def _get_matching_activations(float_dict, quantized_dict):
+    r"""Find the matching activation between float and quantized dict.
+
+    Args:
+        float_dict: recording observer dict of the float model
+        quantized_dict: recording observer dict of the quantized model
+
+    Return:
+        act_dict: dict with key corresponding to quantized module names and each
+        entry being a dictionary with two keys 'float' and 'quantized', containing
+        the matching float and quantized activations
+    """
+    act_dict = {}
+    for key in quantized_dict:
+        match_key = _find_match(sorted(float_dict, reverse=True), key, "stats")
+        if match_key is not None:
+            act_dict[key] = {}
+            act_dict[key]["float"] = float_dict[match_key]
+            act_dict[key]["quantized"] = quantized_dict[key]
+    return act_dict
+
+
+def compare_model_outputs(float_model, q_model, data, white_list=None):
+    r"""Returns a dict with key corresponding to quantized module names and each
+    entry being a dictionary with two keys 'float' and 'quantized', containing
+    the activations of quantized model and float model at matching locations. This
+    dict can be used to compare and compute the propagation quantization error.
+
+    Args:
+        float_module: the float module used to generate the q_module
+        q_module: the quantized module
+        data: input data
+        white_list: list of modules want to compare
+
+    Return:
+        act_compare_dict: dict with key corresponding to quantized module names
+        and each entry being a dictionary with two keys 'float' and 'quantized',
+        containing the matching float and quantized activations
+    """
+    if white_list is None:
```

Comment:
Suggest creating a default white_list in https://github.com/pytorch/pytorch/blob/master/torch/quantization/default_mappings.py and using that as a default argument to compare_model_outputs